src/App.js

```diff
@@ -1,14 +1,24 @@
 import React from 'react';
+import {
+  BrowserRouter, Routes, Route,
+} from 'react-router-dom';
 import './App.css';
-import Calculator from './components/Calculator';
-import Quote from './components/Quote';
+import Calculator from './pages/Calculator';
+import Quote from './pages/Quote';
+import Home from './pages/Home';
+import Navbar from './components/Navbar';
 
 const App = () => (
-  <div className="App">
-    <link href="https://fonts.cdnfonts.com/css/seven-segment" rel="stylesheet" />
-    <Calculator />
-    <Quote />
-  </div>
+  <BrowserRouter>
+    <Routes>
+      <Route path="/" element={<Navbar />}>
+        <Route index element={<Home />} />
+        <Route path="calculator" element={<Calculator />} />
+        <Route path="quotes" element={<Quote />} />
+      </Route>
+    </Routes>
```

Comment:
- [ ] In accordance to the project requirement, kindly ensure that you design the three pages as expected. I believe that ability to reproduce a given wireframe perfectly shows how well we give attention to details. You need to adjust the margins, proper placement of element for the calculator page. 👍 

- [ ] More so, kindly improve the look and feel also as stated in the requirement. 👍 

## Wireframe
<img width="622" alt="Untitled" src="https://github.com/danifromecuador/react-calculator/assets/103386183/ddb45501-ff99-426a-88b1-2a30bad8cb05">